packages/blade/src/components/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -296,12 +296,14 @@ const CommonPage = (): React.ReactElement => {
 
         <Text marginBottom="spacing.3">Trigger URL Change:</Text>
         <Box display="flex" flexDirection="column" gap="spacing.2">
-          <BreadcrumbNavLink to={urls.home}>Home</BreadcrumbNavLink>
-          <BreadcrumbNavLink to={urls.products}>Products</BreadcrumbNavLink>
-          <BreadcrumbNavLink to={urls.payments}>Payments</BreadcrumbNavLink>
-          <BreadcrumbNavLink to={urls.intPayments}>International Payments</BreadcrumbNavLink>
+          <BreadcrumbNavLink to={urls.home}>{'1.1'} Home</BreadcrumbNavLink>
+          <BreadcrumbNavLink to={urls.products}>{'1.1'} Products</BreadcrumbNavLink>
+          <BreadcrumbNavLink to={urls.payments}>{'1.1'} Payments</BreadcrumbNavLink>
+          <BreadcrumbNavLink to={urls.intPayments}>
+            {'1.1.2'} International Payments
```

Comment:
what is this change 😕 ?

Reply:
Offtopic: Why does it have line breaks? Shouldn't it look like breadcrump?
<img width="374" alt="image" src="https://github.com/razorpay/blade/assets/30949385/e6a011bb-b753-46bc-91a9-d99bd141122f">

Reply:
Oh my bad! Understood 🤦🏼